projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="menu">
+  <nb-card *ngIf="!hideUser" [routerLink]="['/users', lab.user.username]" class="user clickable">
+    <nb-user [name]="lab.user.name" [picture]="lab.user.avatar" [title]="lab.user.designation"></nb-user>
+  </nb-card>
+  <nb-card *ngIf="!hideSteps" class="steps">
+    <nb-list>
+      <nb-list-item (click)="onSetStep(-1)" [ngClass]="{'active': selectedLabStep === -1}"
+                    [routerLink]="['/labs', lab.slug]"
+                    class="clickable">
+        <a [routerLink]="['/labs', lab.slug]">Overview</a>
+      </nb-list-item>
+      <nb-list-item (click)="onSetStep(i)" *ngFor="let step of lab.lab_steps; let i = index"
+                    [ngClass]="{'active': selectedLabStep === i}" [routerLink]="['steps', lab.lab_steps[i].id]"
+                    class="clickable">
+        <div>
+          <a [routerLink]="['steps', lab.lab_steps[i].id]">
+            <div>
+              {{step.name}}
+            </div>
+          </a>
+        </div>
+      </nb-list-item>
+    </nb-list>
+  </nb-card>
+  <nb-card *ngIf="similarLabs.length > 0 && !hideRelatedLabs" class="related-labs">
```

Comment:
let's hide this using just css by comparing the window width

Reply:
Since I'm using the same component in two different places and displaying two different things in those two places, it wouldn't be possible with comparing window width right?